console/src/components/Services/Data/RawSQL/RawSQL.js

```diff
@@ -552,6 +523,7 @@ const mapStateToProps = state => ({
   currentSchema: state.tables.currentSchema,
   allSchemas: state.tables.allSchemas,
   serverVersion: state.main.serverVersion ? state.main.serverVersion : '',
+  location: state.routing.locationBeforeTransitions,
```

Comment:
```suggestion
```

Reply:
Good catch! This was the first approach!

Reply:
Done